AutoMapper.Data.Tests/DataReaderMapping.cs

```diff
@@ -395,9 +401,8 @@ public IDataReader BuildDataReaderWithNullableField()
         public When_mapping_a_data_reader_to_a_dto_with_nullable_enum()
         {
             Mapper.Initialize(cfg => {
-                MapperRegistry.Mappers.Insert(0, new DataReaderMapper());
-
-                cfg.CreateMap<IDataReader, DtoWithSingleNullableField>();
+                cfg.Mappers.Insert(0, new DataReaderMapper());
+                cfg.AddMemberConfiguration().AddMember<DataRecordMemberConfiguration>();

```

Comment:
Cool.  This is the first time I've seen someone use `ISourceToDestinationNameMapper` or `IChildMemberConfiguration` since I created them way back when.